domain/src/test/java/org/oppia/android/domain/survey/SurveyGatingControllerTest.kt

```diff
@@ -280,6 +286,7 @@ class SurveyGatingControllerTest {
   }
 
   @Test
+  @EnableFeatureFlag(FeatureFlag.ENABLE_NPS_SURVEY)
```

Comment:
Since this test class is valid  only for the feature flag enabled situation, I think it is sufficient to enable the flag at class level.